wolox-books/src/app/components/auth/auth.component.scss

```diff
@@ -0,0 +1,27 @@
+@import 'src/scss/color';
+@import 'src/scss/fonts';
+
+.container-nav {
+  align-items: center;
+  background-color: $wild-sand;
+  border-top: 5px solid $cerulean;
+  display: flex;
+  font-family: 'Roboto', sans-serif;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 55px;
+  width: 100%;
+}
+
+.logo-wolox {
+  object-fit: cover;
+  padding-left: 100px;
```

Comment:
Remove padding here, add it in the parent component, you are trying to add space between parent component (Container) and children.